src/app/content/components/TableOfContents/index.tsx

```diff
@@ -23,28 +24,185 @@ interface SidebarProps {
   page?: Page;
 }
 
+function TabTrapper({
```

Comment:
is there a reason for this to be a component and not just a hook?

Reply:
Conditional execution. [Check for mobile](https://github.com/openstax/rex-web/pull/2102/files#diff-26081ab52cd6866f337549a6aa3caad9d12eb13241e23c921b76e6f51c746b35R34) fails if window is undefined.